src/main/java/org/jabref/gui/Base.css

```diff
@@ -649,15 +649,27 @@ TextFlow > .tooltip-text-monospaced {
 }
 
 .table-row-cell:matching-search-and-groups {
-    -fx-background-color: white;
+    -fx-background-color: #FFFFFFFF;
```

Comment:
Please either make them derived or add some new constants, as this makes it easier for dark theme to change the colors